docs/docsite/rst/dev_guide/developing_plugins.rst

```diff
@@ -73,6 +73,12 @@ To access the configuration settings in your plugin, use ``self.get_option(<opti
 
 Plugins that support embedded documentation (see :ref:`ansible-doc` for the list) must include well-formed doc strings to be considered for merge into the Ansible repo. If you inherit from a plugin, you must document the options it takes, either via a documentation fragment or as a copy. See :ref:`module_documenting` for more information on correct documentation. Thorough documentation is a good idea even if you're developing a plugin for local use.
 
+Plugins also support 2 additional options not documented in the `options` section in modules.
```

Comment:
more than 2

Reply:
cli, keywords(but might deprecate) and (currently unused) yaml

Reply:
Also, we need to add a note about 'vars', they are not available for all plugins, even when available, it might not mean all vars, for example inventory plugins have mostly just extra_vars available (aside from inventory being additive, but that is for advanced authors only).

Reply:
I have no idea if this is still relevant.  Unless someone wants to take over the PR and make needed edits, please close